src/blocktypes/proviso/publish.sh

```diff
@@ -0,0 +1,29 @@
+SKOPEO=${SKOPEO:-$(dirname "${BASH_SOURCE[0]}")/skopeo.sh}
+
+function proviso() {
+  local -n input=$1
+  local -n output=$2
+
+  local missing tag
+  local -a images filtered
+
+  missing=$(mktemp)
+  trap 'rm -f $missing' RETURN
+
+  mapfile -t images < <(jq -r '.meta.tags[0]|select(.!=null)' <<<"${input[@]}")
```

Comment:
Is that really plural? I don't fully understand the jq foo but `tags[0]` looks like a `head` operation that would result with a singleton.

Reply:
yes we only want to check the first tag, that's explicit (see update PR description).

Reply:
I mean: is it still `images` (plural), though?

Reply:
yeah because we take the first tag over all the input images. Remember, the proviso runs over all the inputs not just a single one.

Reply:
Ah yeah! Click. :smile: